APHKLogicExtractor/DataModel/ApplicationInput/ApplicationInput.cs

```diff
@@ -14,19 +14,14 @@ public enum InputType
     WorldDefinition,
 }
 
-public record ApplicationInput
+internal record ApplicationInput
 {
-    [JsonProperty]
-    internal InputType type;
+    public InputType Type { get; set; }
     // JsonLogic => JsonLogicConfiguration
     // RandoContext => JToken
     // WorldDefinition => StringWorldDefinition
-    [JsonProperty]
-    internal MaybeFile<JToken> configuration;
-    [JsonProperty]
-    internal string? startStateTerm;
-    [JsonProperty]
-    internal MaybeFile<StateClassificationModel>? classifierModel;
-    [JsonProperty]
-    internal MaybeFile<HashSet<string>>? emptyRegionsToKeep;
+    public MaybeFile<JToken> Configuration { get; set; }
```

Comment:
I feel like this would give nullability warnings as written, does it need to be `required` (tells the compiler it will be nonnull because it is expected to be assigned during initialization)?

Reply:
Yeah, there was a few of those warnings, I ended up fixing them.